src/containers/images.js

```
@@ -0,0 +1,108 @@
+import React, {
+  Component
+} from 'react';
+import sha1 from 'sha1';
+import superagent from 'superagent';
+
+import {
+  connect
+} from 'react-redux';
+
+import {
+  updateEmployee
+} from '../actions/index';
+import ImageDrop from '../components/imageDrop';
+
+
+class Images extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      hover: false
+    };
+
+    this.uploadFile = this.uploadFile.bind(this);
+    this.toggleHover = this.toggleHover.bind(this);
+  }
+
+  uploadFile(files) {
+    const image = files[0];
+    const cloudName = 'blank';
+    const url = `https://api.cloudinary.com/v1_1/${cloudName}/image/upload`;
+
+    const timestamp = Date.now() / 1000;
+    const uploadPreset = 'ansattliste';
+
+    const publicId = this.props.employee.data.id;
+
+    const paramStr = `public_id=${publicId}\
+&timestamp=${timestamp}\
+&upload_preset=${uploadPreset}\
+MxK1OBDt-H488-5dUMB64sJb8NY`;
+
+    const signature = sha1(paramStr);
+
+    const params = {
+      api_key: '295669173729256',
+      public_id: publicId,
+      timestamp,
+      upload_preset: uploadPreset,
+      signature,
+    };
+
+    const uploadRequest = superagent.post(url);
+    uploadRequest.attach('file', image);
+
+    Object.keys(params).forEach((key) => {
+      uploadRequest.field(key, params[key]);
+    });
+
+    uploadRequest.end((err, resp) => {
+      if (err) {
+        return;
```

Perhaps it would be better to report this error? Or is it obvious that it failed?